models/userModel.js

```diff
@@ -17,6 +17,7 @@ const userSchema = new mongoose.Schema({
   history: {
     type: [historyItemSchema],
   },
+  // add dollar balance and bitcoin balance
```

Comment:
@Klutch27 Did we decide we want these as a part of the user schema instead of /in addition to the history?